common/src/main/scala/org/apache/spark/sql/comet/parquet/CometParquetReadSupport.scala

```diff
@@ -369,7 +363,7 @@ object CometParquetReadSupport {
   /**
    * Whether the parquet schema contains any field IDs.
    */
-  private def containsFieldIds(schema: Type): Boolean = schema match {
+  def containsFieldIds(schema: Type): Boolean = schema match {
```

Comment:
Any reason that this is now public?

Reply:
The corresponding function is not private in Spark either, but you're right, we do not need it to be public at the moment though we might need to for better field id support. Changed it back to private.